docs/deployment/rds-s3/auto-rds-s3-setup.py

```diff
@@ -545,10 +550,19 @@ def get_pipeline_params_env_lines(pipeline_params_env_file):
     return pipeline_params_env_lines
 
 
-def get_updated_pipeline_params_env_lines(db_instance_info, pipeline_params_env_lines):
+def get_updated_pipeline_rds_params_env_lines(db_instance_info, pipeline_params_env_lines):
     db_host_pattern = "dbHost="
     db_host_new_line = db_host_pattern + db_instance_info["Endpoint"]["Address"] + "\n"
 
+    new_pipeline_params_env_lines = []
+
+    for line in pipeline_params_env_lines:
+        line = replace_line(line, db_host_pattern, db_host_new_line)
+        new_pipeline_params_env_lines.append(line)
+
+    return new_pipeline_params_env_lines
```

Comment:
I had to add one more `replace_line` for a different pattern within the for loop. Not sure how to do that with list comprehension.

Reply:
```
    new_pipeline_params_env_lines = []

    for line in pipeline_params_env_lines:
        line = replace_line(line, db_host_pattern, db_host_new_line)
        line = replace_line(line, db_mlmd_db_pattern, db_mlmd_db_new_line)
        new_pipeline_params_env_lines.append(line)

    return new_pipeline_params_env_lines
```

Reply:
In this case it would be less prettier and concise
```python
    return [
        replace_line (
            replace_line(line, db_host_pattern, db_host_new_line),
            db_mlmd_db_pattern, db_mlmd_db_new_line)
        for line in pipeline_params_env_lines
    ]
```